GVFS/GVFS.Virtualization/FileSystemCallbacks.cs

```diff
@@ -296,6 +296,57 @@ public EventMetadata GetAndResetHeartBeatMetadata(out bool logToFile)
             return metadata;
         }
 
+        public bool DehydrateFolder(string relativePath)

```

Comment:
This is close to a `Try...` patterned method. If we use a `Try` prefix, it would at least indicate to callers they need to check the result.

Reply:
The modified paths has a `TryAdd` so it shouldn't throw but the placeholder add could so added a try/catch around it and renamed the method to `TryDehydrateFolder`